CHANGELOG.md

```diff
@@ -1,13 +1,15 @@
 v4.1.6
```

Comment:
In order to release this, you need to add another section above this as v4.1.7

Reply:
done